utils/labels.cc

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright (C) 2022-present ScyllaDB
+ */
+
+/*
+ * SPDX-License-Identifier: AGPL-3.0-or-later
+ */
+
+#include "utils/labels.hh"
+
+seastar::metrics::label level_label("level");
```

Comment:
They do sent to promethues, it's a bit of a waste, that's true, but it is still useful other systems (i.e Datadog) will be able to query based on those labels.

I've mentioned it in the PR header, there is an additional functionality that I would like to add, that would allow to configure (first from the configuration, then from the API) specific labeling, with that extra configuration I can add functionality to drop labels before sending.